common/fs/src/cache/mod.rs

```diff
@@ -673,8 +676,12 @@ impl FileSystem {
         debug!("inserting {:?}", path);
 
         if !self.passes(path, _entries) {
-            info!("ignoring {:?}", path);
-            return Ok(None);
+            // Do not continuously log ignored files
+            if let false = self.ignored_dirs.contains(path) {
+                self.ignored_dirs.insert(path.to_path_buf());
+                info!("ignoring {:?}", path);
+                return Ok(None);
```

Comment:
We should still return early, even if we've seen it before, so we should move this return back down a line

Reply:
Good catch! Fixed.